server/protocols/jmap-rfc-8621/src/main/scala/org/apache/james/jmap/http/SessionSupplier.scala

```diff
@@ -0,0 +1,76 @@
+/** **************************************************************
+ * Licensed to the Apache Software Foundation (ASF) under one   *
+ * or more contributor license agreements.  See the NOTICE file *
+ * distributed with this work for additional information        *
+ * regarding copyright ownership.  The ASF licenses this file   *
+ * to you under the Apache License, Version 2.0 (the            *
+ * "License"); you may not use this file except in compliance   *
+ * with the License.  You may obtain a copy of the License at   *
+ * *
+ * http://www.apache.org/licenses/LICENSE-2.0                 *
+ * *
+ * Unless required by applicable law or agreed to in writing,   *
+ * software distributed under the License is distributed on an  *
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY       *
+ * KIND, either express or implied.  See the License for the    *
+ * specific language governing permissions and limitations      *
+ * under the License.                                           *
+ * ***************************************************************/
+
+package org.apache.james.jmap.http
+
+import org.apache.james.core.Username
+import org.apache.james.jmap.model._
+
+object SessionSupplier {
+  private val MAX_SIZE_UPLOAD = UnsignedInt(50_000_000)
+  private val MAX_CONCURRENT_UPLOAD = UnsignedInt(8)
+  private val MAX_SIZE_REQUEST = UnsignedInt(10_000_000)
+  private val MAX_CONCURRENT_REQUESTS = UnsignedInt(8)
+  private val MAX_CALLS_IN_REQUEST = UnsignedInt(32)
+  private val MAX_OBJECTS_IN_GET = UnsignedInt(256)
+  private val MAX_OBJECTS_IN_SET = UnsignedInt(128)
+  private val URL = "http://this-url-is-hardcoded.org"
+  private val STATE = "thisIsHardCodedState"
+  private val ACCOUNT_ID = Id("thisIsHardCodedId")
```

Comment:
Seems like we don't have UserId, in the other hand, username contains '@' special character that cannot make it become an accountId

Reply:
That's a real issue.

We should discuss the design aspect of this....

Solution that I can think of:
  - Having a AccountRepository (UUID -> Account - including email)
  - Using an escaping strategy

I personally do think we should go for the first one. We might need to modify Sprint content to make this fit in.

Reply:
In a temporary fashion we might consider user thehash of the mail address...

Reply:
did you create an issue for this AccountId improvement?

Reply:
created in sprint 17 backlog